internal/versiongen/gen.go

```diff
@@ -9,23 +9,25 @@ import (
 	"encoding/json"
 	"flag"
 	"fmt"
-	"io/ioutil"
+	"io"
 	"log"
 	"net/url"
 	"os"
 	"sort"
 	"text/template"
-	"time"
 
 	"github.com/hashicorp/go-cleanhttp"
 	"github.com/hashicorp/go-version"
 )
 
-var baseURL = "https://api.releases.hashicorp.com/v1"
+var baseURL = "https://get.opentofu.org"
 
 type release struct {
-	Version *version.Version `json:"version"`
-	Created *time.Time       `json:"timestamp_created"`
+	Version *version.Version `json:"id"`
+}
+
+type versionsAPI struct {
+	Versions []release
```

Comment:
```suggestion
	Versions []release `json:"versions"`
```
Recommended to have the json tags too, even if the struct field name is matching the expected json field.